Code/training_common_utils.py

```diff
@@ -1,25 +1,27 @@
 import torch
 import torch.nn as nn
 import time
+import pandas as pd
 import numpy as np
-from CheXpertDataset import ChexpertDataset, UZerosTransform, UOnesTransform, ReplaceNaNTransform
+from CheXpertDataset import ChexpertDataset, UZerosTransform, UOnesTransform, ReplaceNaNTransform, GreyScale_to_RGB
 from torchvision import transforms
 from torch.utils.data import Subset, DataLoader
 
 image_preprocessing = transforms.Compose([
+    # GreyScale_to_RGB(),
     transforms.Resize((600, 600)),
     transforms.ToTensor(),
-    transforms.Normalize((0.0, 0.0, 0.0), (1.0, 1.0, 1.0))
+    transforms.Normalize((0, 0, 0), (1.0, 1.0, 1.0))
 ])
 
 label_preprocessing_uzeros = transforms.Compose([
     UZerosTransform(),
-    ReplaceNaNTransform()
+    ReplaceNaNTransform(),
 ])
 
 label_preprocessing_uones = transforms.Compose([
     UOnesTransform(),
-    ReplaceNaNTransform()
+    ReplaceNaNTransform(),
```

Comment:
Just so we don't forget
```suggestion
    ReplaceNaNTransform()
```